### PR TITLE
Add stats to profile layout

### LINKED ISSUE #55: Feedback: Dog Owner Profile Info
TA says:
> The dog owner's profile page may also include a bit more information, such as past meeting history, related stats, etc.

### PR DESCRIPTION
**Pull request by divy4**
## Profile Page
screenshot_1544024988

## Review Page (Changed time format)
screenshot_1544025139

Edit: Fixes #55 (maybe if I put this here it'll show up in the issue...)

### CODE REVIEW

File: app/src/main/res/layout/activity_dog_owner_profile.xml [hunk 1]
```diff
@@ -25,12 +25,175 @@
...
+        android:layout_marginTop="20dp">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/stats"
+            android:textSize="24sp"
+            android:textAlignment="center"/>
+
+        <LinearLayout
```
Comment (divy4) [inline app/src/main/res/layout/activity_dog_owner_profile.xml]: Would've loved to just use a grid layout here. But apparently grid layout doesn't support evenly distributing column width.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -21,6 +21,8 @@
     <string name="create_meetup_social_level_description">Select how many people you would like to attend your meetup. Although we can\'t guarantee the minimum number people will show up, we\'ll do our best to make sure no more than the maximum will.</string>
     <string name="create_meetup_photo_title"><u>Dog Snapshot</u></string>
     <string name="dogs">Dogs:</string>
+    <string name="dogs_petted">Dogs petted:</string>
```
[the remark this answers is not in the document]
Comment (divy4) [reply app/src/main/res/values/strings.xml]: Just did a google search, looks like petted wins. [https://writingexplained.org/pet-vs-petted](https://writingexplained.org/pet-vs-petted)

File: app/src/main/java/cs465/illinois/edu/dogonthequad/DataModels/User.java [hunk 3]
```diff
@@ -1,24 +1,46 @@
...
+    public Integer mMeetupsAttended;
+    public Integer mDogsPetted;
+    public Integer mPeopleMet;
 
     public User() {
         this.mId = UUID.randomUUID();
         this.mName = "";
         this.mDogs = new ArrayList<UUID>();
+        this.mUserSince = new Date();
+        this.mMeetupsCreated = 5;
```
[the remark this answers is not in the document]
Comment (divy4) [reply app/src/main/java/cs465/illinois/edu/dogonthequad/DataModels/User.java]: Yeah, I figured I should do that. But I'm not sure how. Do I just add the key-values to the JSON's?

[the remark this answers is not in the document]
Comment (schmidtwmark) [reply app/src/main/java/cs465/illinois/edu/dogonthequad/DataModels/User.java]: Just merge it and I'll get it done, easier than explaining it

**Pull request merged**